src/DevCenter/DevCenter.AutoRest/README.md

```diff
@@ -0,0 +1,123 @@
+<!-- region Generated -->
+# Az.DevCenter
+This directory contains the PowerShell module for the DevCenter service.
+
+---
+## Status
+[![Az.DevCenter](https://img.shields.io/powershellgallery/v/Az.DevCenter.svg?style=flat-square&label=Az.DevCenter "Az.DevCenter")](https://www.powershellgallery.com/packages/Az.DevCenter/)
+
+## Info
+- Modifiable: yes
+- Generated: all
+- Committed: yes
+- Packaged: yes
+
+---
+## Detail
+This module was primarily generated via [AutoRest](https://github.com/Azure/autorest) using the [PowerShell](https://github.com/Azure/autorest.powershell) extension.
+
+## Module Requirements
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 2.7.5 or greater
+
+## Authentication
+AutoRest does not generate authentication code for the module. Authentication is handled via Az.Accounts by altering the HTTP payload before it is sent.
+
+## Development
+For information on how to develop for `Az.DevCenter`, see [how-to.md](how-to.md).
+<!-- endregion -->
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+```yaml
+# pin the swagger version by using the commit id instead of branch name
+branch: 4f6418dca8c15697489bbe6f855558bb79ca5bf5
+require:
+# readme.azure.noprofile.md is the common configuration file
+  - $(this-folder)/../../readme.azure.noprofile.md
+input-file:
+  - $(this-folder)/../../specification/devcenter/resource-manager/Microsoft.DevCenter/stable/2023-04-01/commonDefinitions.json
```

Comment:
The reason why build failed is due to the incorrect swagger file URLs. Please replace them all in this README.md with the following format.
```suggestion
  - $(repo)/specification/devcenter/resource-manager/Microsoft.DevCenter/stable/2023-04-01/commonDefinitions.json
```

Reply:
@vidai-msft I made updates to the swagger files due to errors with some LROs from the autorest generated cmdlets.(Needed to have the 200 OK response not just 202 accepted response in the swagger file). Will there be issues if these changes are not checked into the azure-rest-api-specs repository?

Reply:
@am-lim AutoRest generated code depends on the content of the swagger files. So different content will generate different code. If you need to update the swagger, I recommend re-generating after the change is merge.

Reply:
@vidai-msft Just to confirm, do I need to check in my changes into the azure-rest-api-specs repository? If yes, is there a way to bypass this process? 

More context: For several of our LROs, the original AutoRest generated code led to an error "OK is not an accepted response". I had to add a 200 response for some of the REST APIs, not just the "Accepted" response.